source/AnimationDebug.hx

```diff
@@ -81,6 +81,8 @@ class AnimationDebug extends FlxState
 
 		genBoyOffsets();
 
+    addHelpText();
```

Comment:
what's with your additions missing one indent?

Reply:
No idea. #2253 (now in master) added the hxformat configuration that automatically manages indentation on file save.